cognite/neat/workflows/steps_registry.py

```diff
@@ -44,8 +49,10 @@ def load_step_classes(self):
             if inspect.isclass(step_cls):
                 logging.info(f"Loading NEAT step {name}")
                 self._step_classes.append(step_cls)
+        sys.path.append(str(Path(self.data_store_path) / "workflows"))
         try:
             if self.user_steps_path:
+                sys.path.append(str(self.user_steps_path))
```

Comment:
Do you need to append them to path to load the custom module?

Reply:
yes that is needed for loading custom steps (modules)